[IMP] Purchase: Reordering Rules Update (#12112)
odoo/documentation: content/applications/inventory_and_mrp/purchase/products/reordering.rst

> falls below a set threshold.

Reviewer: This intro is written in a passive tone, and isn't direct about  why reordering rules are helpful to the purchasing process to begin with.

Let's address:
- Why do businesses need to keep minimum amount available in stock?
- What are common challenges or inefficiencies of keeping things in stock?

And reordering rules make it easier for companies because:
- Keep highly demanded products in-stock at all times using reordering rules, that trigger a RFQ each time the forecasted stock quantities fall below the minimum.
- RFQs generated from reordering rules have the vendor, price, quantity needed to reorder, which makes things faster and more convenient (:

   .. example::
      If :guilabel:`Max Quantity` is set to `100` but :guilabel:`Multiple Quantity` is set to order
      the product in batches of `200`, a purchase order is then created for 200 units of the
      product.

- :guilabel:`Unit` specifies the unit of measurement by which the quantity is to be ordered. For
  discrete products, this should be set to `Units`. However, it can also be set to units of
  measurement like `Volume` or `Weight` for non-discrete products like water or bricks.

.. image:: reordering/reordering-rule-configuration.png
   :alt: Configure the reordering rule in Odoo.

Manually trigger reordering rules using the scheduler
=====================================================

Reordering rules are automatically triggered by the scheduler, which runs once a day by default. To

> trigger reordering rules manually, navigate to :menuselection:`Inventory app --> Operations -->

Reviewer: ```suggestion
trigger reordering rules manually, turn on developer mode, navigate to :menuselection:`Inventory app --> Operations -->
```

   .. example::
      If :guilabel:`Max Quantity` is set to `100` but :guilabel:`Multiple Quantity` is set to order
      the product in batches of `200`, a purchase order is then created for 200 units of the
      product.

- :guilabel:`Unit` specifies the unit of measurement by which the quantity is to be ordered. For
  discrete products, this should be set to `Units`. However, it can also be set to units of
  measurement like `Volume` or `Weight` for non-discrete products like water or bricks.

.. image:: reordering/reordering-rule-configuration.png
   :alt: Configure the reordering rule in Odoo.

.. seealso::
   :doc:`../../inventory/warehouses_storage/replenishment/reordering_rules`

Manually trigger reordering rules using the scheduler
=====================================================

Reordering rules are automatically triggered by the scheduler, which runs once a day by default. To
trigger reordering rules manually, turn on developer mode, navigate to :menuselection:`Inventory app

> --> Operations --> rocurement: Run Scheduler`. On the pop-up window, confirm the manual action by

Reviewer 2: Typo :)
```suggestion
--> Operations --> Procurement: Run Scheduler`. On the pop-up window, confirm the manual action by
```

To manage the reordering rules for a single product, navigate to that product page's form and select

> the :guilabel:`Reordering Rules` tab at the top of the form.

Reviewer: ```suggestion
the :guilabel:`Reordering Rules` smart button at the top of the form.
```
